data_server.py

```diff
@@ -0,0 +1,62 @@
+#Module to load dataset and divide it into train, test and validation sets for X and Y
+
+def Load_dataset(filename):
+
+    import numpy as np
+
+    try:
+        file = open(filename)
+    except:
+        print("Unable to open the file")
```

Comment:
Is it needed? You basically remove the useful exception message and replace it with ambiguous "Unable to open the file".